.changeset/witty-bats-drum.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": minor
+---
+
+remove bundle size warning
```

Comment:
fix: remove bundle size warning from Worker deploy commands

Bundle size was a proxy for startup time. Now that we have startup time
reported, focus on bundle size is less relevant.